pom.xml

```diff
@@ -313,12 +313,7 @@
                     <plugin>
                         <groupId>com.qulice</groupId>
                         <artifactId>qulice-maven-plugin</artifactId>
-                         <!--
-                         @todo #558:30min Upgrade Qulice to 0.16. Fix all checkstyle
-                          violations (mostly `AtclauseOrderCheck`). Fix or add puzzle for
-                          PMD, findbugs and other types of violations.
-                         -->
-                        <version>0.15.4</version>
+                        <version>0.16</version>
```

Comment:
@essobedo I think in scope of this ticket we should get rid of a few of the lines below. otherwise, your contribution is very small